docs/getting-started-guides/ubernetes-lite.md

```diff
@@ -0,0 +1,190 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+Create the first cluster, but pass MULTIZONE to tell the cluster to manage multiple zones; creating nodes in us-central1-a.
+
+```
+MULTIZONE=1 KUBE_GCE_ZONE=us-central1-a cluster/kube-up.sh
+```
+
+View the nodes; you can see that they are tagged with the zone they are in (they are all in us-central1-a so far):
+
+```
+kubectl get nodes
+```
+
+Add more nodes to the existing cluster, reusing the existing master, running in a different zone (us-central1-b):
+
+```
+KUBE_USE_EXISTING_MASTER=true MULTIZONE=1 KUBE_GCE_ZONE=us-central1-b cluster/kube-up.sh
+```
+
+View the nodes again; 3 more nodes should have launched and be tagged in us-central1-b:
+
+```
+kubectl get nodes
+```
+
+Create a volume (only PersistentVolumes are supported for zone affinity):
+
+```
+gcloud compute disks create --size=5GB --zone=us-central1-a my-data-disk
+
+kubectl create -f - <<EOF
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: my-data-disk
+spec:
+  capacity:
+    storage: "5Gi"
+  accessModes:
+    - "ReadWriteOnce"
+  gcePersistentDisk:
+    pdName: "my-data-disk"
+    fsType: "ext4"
+EOF
+```
+
+The PV is also tagged with the zone & region is is in (us-central1-a):
+
+```
+> kubectl get pv
+NAME           LABELS                                                                                                        CAPACITY   ACCESSMODES   STATUS      CLAIM     REASON    AGE
+my-data-disk   failure-domain.alpha.kubernetes.io/region=us-central1,failure-domain.alpha.kubernetes.io/zone=us-central1-a   5Gi        RWO           Available                       5s
```

Comment:
This has been changed from alpha to beta.

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/unversioned/well_known_labels.go#L20